blaze-client/src/main/scala/org/http4s/client/blaze/BlazeClientBuilder.scala

```diff
@@ -112,21 +113,20 @@ sealed abstract class BlazeClientBuilder[F[_]] private (
 
   /** Use the provided `SSLContext` when making secure calls */
   def withSslContext(sslContext: SSLContext): BlazeClientBuilder[F] =
-    copy(sslContext = Some(sslContext))
+    withSslContextOption(Some(sslContext))
 
   /** Use an `SSLContext` obtained by `SSLContext.getDefault()` when making secure calls.
     *
-    * The creation of the context is lazy, as `SSLContext.getDefault()` throws on some
-    * platforms.  The context creation is deferred until the first secure client call.
+    * Since 0.21, the creation is not deferred.
     */
   def withDefaultSslContext: BlazeClientBuilder[F] =
-    copy(sslContext = None)
+    withSslContext(SSLContext.getDefault())
 
-  @deprecated("Use `withSslContext` or `withDefaultSslContext`", "0.20.2")
+  /** Use some provided `SSLContext` when making secure calls, or disable secure calls with `None` */
   def withSslContextOption(sslContext: Option[SSLContext]): BlazeClientBuilder[F] =
     copy(sslContext = sslContext)
 
-  @deprecated("Use `withDefaultSslContext`", "0.20.2")
+  /** Disable secure calls */
   def withoutSslContext: BlazeClientBuilder[F] =
     withDefaultSslContext
```

Comment:
should this be `copy(sslContext = None)`?

Reply:
You are absolutely right!  Good catch.